### DISCUSSION TITLE
How to get a sense for routing congestion heat map for floorplan

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD-flow-scripts#768] [General] [answered]
Any advice on how to a get sense for how well a floor plan is laid out w.r.t. routing congestion?

I'm thinking a heatmap which shows reflects overlap for direct shortest routes disregarding conflicts, which should be fast to calculate, would be helpful. If I had the ability to zoom in and discern routes and nets, I would be able to figure out what I could do differently at the RTL stage or floorplan.

It is possible to iterate very quickly on the floorplan, if the PDN step is skipped.

**Comment by maliberty** [comment, marked as answer]
after global routing you can see the congestion map in the GUI today
